docker/dotcms/Dockerfile

```diff
@@ -14,71 +15,90 @@ ARG BUILD_ID=HEAD
 
 WORKDIR /srv
 
+# Set gradle options
+ENV GRADLE_USER_HOME=/build/src/core/dotCMS/.gradle
+ENV GRADLE_OPTS="-Dfile.encoding=utf-8 -Xmx2g"
+
+
 # dotCMS core distributed under GPLv3 license (https://github.com/dotCMS/core/blob/master/license.txt)
 COPY build-src/ /build/
 RUN chmod 500 /build/build_dotcms.sh && /build/build_dotcms.sh ${BUILD_FROM} ${BUILD_ID}
```

Comment:
One would think that this script is pointless if we use what we currently have in the project.
Project, not repo since it will require for changes to be in sync with remote. I mean, our docker image completely relies on the git repo, if that remains to be our strategy we will always use this script, which is find by me.

On the other hand we have the Dockerfile already in `core` repo, and it might require some file relocation in the project, but we can just copy whatever it is in there (even untracked or staged files) to the docker image when building. I guess it will be useful for development purposes to not build based on git. If this is the case we can get rid of of `dotcms-seed`.

I guess it's up to our needs. To me it seems fine what we currently have. And this multistage strategy you propose decouples java installation (through sdkman isolated image) from actual dotcms files handling.